Week1/homework/app.js

```diff
@@ -2,10 +2,136 @@
+    `English`,
+    `Kate Wilhelm`,
+  );
+  favoriteBooks[bookTitles[3]] = new FavoriteBook(`To Kill a Mockingbird`, `English`, `Harper Lee`);
+  favoriteBooks[bookTitles[4]] = new FavoriteBook(`The Help`, `English`, `Kathryn Stockett`);
+  favoriteBooks[bookTitles[5]] = new FavoriteBook(`Never Let Me Go`, `English`, `Kazuo Ishiguro`);
+  favoriteBooks[bookTitles[6]] = new FavoriteBook(
+    `Gone With The Wind`,
+    `English`,
+    `Margaret Mitchell`,
+  );
+  favoriteBooks[bookTitles[7]] = new FavoriteBook(`It`, `English`, `Stephan King`);
+  favoriteBooks[bookTitles[8]] = new FavoriteBook(
+    `Interpreter of Maladies`,
+    `English`,
+    `Jhumpa Lahiri`,
+  );
+  favoriteBooks[bookTitles[9]] = new FavoriteBook(`1Q84`, `English`, `Haruki Murakami`);
+
+  function BookCover(coverImage) {
+    this.coverImage = coverImage;
+  }
+
+  // eslint-disable-next-line no-new-object
+  const favoriteBooksCovers = new Object();
+
+  favoriteBooksCovers[bookTitles[0]] = new BookCover(`images/the_road.jpg`);
+  favoriteBooksCovers[bookTitles[1]] = new BookCover(`images/one_hundred_years_of_solitude.jpg`);
+  favoriteBooksCovers[bookTitles[2]] = new BookCover(`images/where_late_the_sweet_birds_sang.jpg`);
+  favoriteBooksCovers[bookTitles[3]] = new BookCover(`images/to_kill_a_mockingbird.jpg`);
+  favoriteBooksCovers[bookTitles[4]] = new BookCover(`images/the_help.jpg`);
+  favoriteBooksCovers[bookTitles[5]] = new BookCover(`images/never_let_me_go.jpg`);
+  favoriteBooksCovers[bookTitles[6]] = new BookCover(`images/gone_with_the_wind.jpg`);
+  favoriteBooksCovers[bookTitles[7]] = new BookCover(`images/it.jpg`);
+  favoriteBooksCovers[bookTitles[8]] = new BookCover(`images/interpreter_of_maladies.jpg`);
+  favoriteBooksCovers[bookTitles[9]] = new BookCover(`images/1q84.jpg`);
+
+  let bookItem;
+  let bookDiv;
+
+  function createBookList() {
+    const bookList = document.createElement('ul');
+
+    for (const key of Object.keys(favoriteBooks)) {
+      bookItem = document.createElement('li');
+      bookDiv = document.createElement(`div`);
+      const bookTitle = document.createElement('h2');
+      const bookAuthor = document.createElement('h4');
+      const bookLang = document.createElement('h4');
+
+      bookItem.className = 'bookItem';
+      bookItem.id = key;
+
+      bookDiv.id = key + `_div`;
+      // addBookCover(bookItem, key);
+      bookTitle.innerText = favoriteBooks[key].title;
+      bookAuthor.innerText = `written by ${favoriteBooks[key].author}`;
+      bookLang.innerText = `available in ${favoriteBooks[key].language}`;
+
+      bookDiv.appendChild(bookTitle);
+      bookDiv.appendChild(bookAuthor);
+      bookDiv.appendChild(bookLang);
+
+      bookItem.appendChild(bookDiv);
+      bookList.appendChild(bookItem);
+    }
+    document.body.appendChild(bookList);
+  }
+
+  createBookList();
+
+  /* At first I added the images to listitems via a function while creating the book list, but when I reread the MAKEME.MD 
```

Comment:
Nice that you learned something from the exercise!

Reply:
Yes, absolutely. Thanks again.